src/app/route/CategoryPage/CategoryPage.container.js

```diff
@@ -323,8 +332,10 @@ export class CategoryPageContainer extends PureComponent {
     }
 
     _requestCategoryWithPageList() {
+        const customFilters = this._getSelectedFiltersFromUrl();
```

Comment:
customFilters are not needed within this function, directly access in a sub-function